02-peer-protocol.md

```diff
@@ -545,6 +545,42 @@ considered to be the channel funder (the sender of `open_channel`).
 The quiescence effect is exactly the same as if one had replied to the
 other.
 
+### `go_on`
+
+1. type: TODO (`go_on`)
```

Comment:
I hope @cdecker can find us a better name for that one ;)

Reply:
Stands for "global operation is over now" 😁

But yeah I'm very open to improvement here 😂

Reply:
`carry_on` 🧐